framework/src/modules/interoperability/base_interoperability_method.ts

```diff
@@ -95,18 +95,18 @@ export abstract class BaseInteroperabilityMethod<
 	private async _getChannelCommon(context: ImmutableMethodContext, chainID: Buffer) {
 		const mainchainID = getMainchainID(chainID);
```

Comment:
I would still move this down below just before it's needed

Reply:
Done